dspace-server-webapp/src/test/data/dspaceFolder/config/item-submission.xml

```diff
@@ -0,0 +1,282 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE item-submission SYSTEM "item-submission.dtd">
+
+<!-- Configurable Submission configuration file -->
+
+<!-- This XML configuration file allows you to configure the ordering -->
+<!-- and number of the steps that occur in the Item Submission Process. -->
+<item-submission>
```

Comment:
Please rebase on the latest `master` as this file already exists on `master`.  So, I suspect a rebase will remove a lot of the changes here and make your PR smaller.

Reply:
We did a rebase on master, but this file already exists in the dspace-api module, but not in the dspace-server-webapp (yet).

Reply:
aha, good point, I was looking in the wrong place! Marking this as resolved.